api/src/business_ar_api/utils/registrar_metadata.py

```diff
@@ -57,6 +57,14 @@ class RegistrarInfo:  # pylint: disable=too-few-public-methods
             "signatureImage": "registrar_signature_5.png",
             "signatureImageAndText": "registrar_signature_and_text_5.png",
             "startDate": "2025-04-18T00:00:00",
+            "endDate": "2025-08-11T23:59:59"
+        },
+        {
+            "name": "KERRY TAYLOR",
+            "title": "Registrar of Companies",
+            "signatureImage": "registrar_signature_6.png",
+            "signatureImageAndText": "registrar_signature_and_text_6.png",
+            "startDate": "2025-08-12T00:00:00",
```

Comment:
Wait! There's a request from the business to defer this. Olga is negotiating the exact requirement. Hence, do not merge this PR yet.